task_1/scss/header.scss

```diff
@@ -0,0 +1,336 @@
+@import './variables';
+
+header {
+  width: 100%;
+  height: auto;
+  background: rgb(9, 9, 121);
+  background: linear-gradient(
+    101deg,
+    rgba(9, 9, 121, 1) 0%,
+    rgba(0, 224, 255, 1) 0%,
+    rgba(34, 191, 86, 1) 100%
+  );
+}
+
+.nav {
+  transition: 0.5s ease;
+  height: 62px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  width: 100%;
+  max-width: 1440px;
+  margin: 0 auto;
+  top: 0;
+  left: 0;
+
+  &-logo {
+    font-weight: 600;
+    font-size: 22px;
+    display: flex;
+  }
+
+  &-list {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    justify-content: space-evenly;
+    position: absolute;
+    top: 63px;
+    left: -100%;
+    transition: 0.7s all ease;
```

Comment:
property name | transition duration | transition style

specify exact properties for transition